app/views/layouts/pdf.html.erb

```diff
@@ -2,6 +2,7 @@
 <html lang="en" class="govuk-template">
   <head>
     <meta charset="utf-8">
+    <title><%= t("layouts.application.header.title") %></title>
```

Comment:
I'd like to UAT this and ensure it isn't doing something weird with the report generator! :+1: